src/config/dynamic-theme-fixes.config

```diff
@@ -13440,6 +13440,24 @@ INVERT
 
 ================================
 
+rte.ie
+
+INVERT
+.masthead .nav-btn.menu
+
+CSS
+.masthead .weather-widget .icon {
+    filter: invert(0);
+}
+.theoplayer-skin .vjs-control-bar {
+    background-color: rgba(34, 41, 50, 0.7);
+}
+.theoplayer-skin .theo-control-bar-shadow {
+    background: #0000001c;
+}
```

Comment:
Could you give me a link where these 2 elements can be found(seems to be from a video player?)

Reply:
The video player might be geo-locked to Ireland but it can be found here https://www.rte.ie/player/onnow